### PR TITLE
Add how users should report security vulnerabilities for this repository

### PR DESCRIPTION
**Pull request by fredriksvantes**
**What type of PR is this?**

Documentation

**What does this PR do? Why is it needed?**

Suggestion to add a notice on how to report security vulnerabilities. This is visible at https://github.com/prysmaticlabs/prysm/security

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+## Supported Versions
+
+Please see [Releases](https://github.com/prysmaticlabs/prysm/releases/). We recommend using the [most recently released version](https://github.com/prysmaticlabs/prysm/releases/latest).
+
+## Reporting a Vulnerability
+
+Please send vulnerability reports to security@chainsafe.io.
```
Comment (nisdas) [inline SECURITY.md:9]: This doesn't seem right :smile: 

```suggestion
Please send vulnerability reports to security@prysmaticlabs.com.
```

Comment (fredriksvantes) [reply SECURITY.md]: Very true, I commited a few other ones and got it mixed up! :)

**Review by nisdas — DISMISSED** [review]
Thanks for this ! Will wait for @prestonvanloon take a look and then we can merge it in.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+## Supported Versions
+
+Please see [Releases](https://github.com/prysmaticlabs/prysm/releases/). We recommend using the [most recently released version](https://github.com/prysmaticlabs/prysm/releases/latest).
+
+## Reporting a Vulnerability
+
+Please send vulnerability reports to security@prysmaticlabs.com.
```
Comment (prestonvanloon) [inline SECURITY.md:9]: Can we add something here about our security.txt file? This is a signed document outlining the preferred encryption and reporting destination.

https://github.com/prysmaticlabs/prysm/blob/develop/.well-known/security.txt

**Review by prestonvanloon — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 2 above)
Comment (prestonvanloon) [reply SECURITY.md:9]: More info on this (draft) standard: https://securitytxt.org/

Comment (fredriksvantes) [reply SECURITY.md]: Yeah that's good, updated it! Have you considered putting it under https://prysmaticlabs.com/.well-known/security.txt as well?
You could also consider this if you wish to further increase visibility: https://dnssecuritytxt.org/

**Review by prestonvanloon — APPROVED** [review]

File: SECURITY.md (hunk 2 above)
Comment (prestonvanloon) [reply SECURITY.md]: Great feedback! We could definitely put that on our website. I'll submit a PR for that later this week.

Regarding the dns security txt, i'll update that immediately. Great suggestion!

Comment (prestonvanloon) [reply SECURITY.md]: DNS is done!
```
;; QUESTION SECTION:
;prysmaticlabs.com.             IN      TXT

;; ANSWER SECTION:
prysmaticlabs.com.      264     IN      TXT     "google-site-verification=JJw5PVDCCqp9N4HsUSghS63qV1Eyy65mlS4OnGY8s6A"
prysmaticlabs.com.      264     IN      TXT     "v=spf1 include:_spf.google.com ~all"
prysmaticlabs.com.      264     IN      TXT     "security_contact=mailto:security@prysmaticlabs.com"
prysmaticlabs.com.      264     IN      TXT     "security_policy=https://github.com/prysmaticlabs/prysm/blob/develop/.well-known/security.txt"
```

**Pull request merged**